pkg/controller/secret/secret_controller.go

```diff
@@ -218,16 +356,15 @@ func getSecretKey(r *ReconcileSecret, request *reconcile.Request, secretname str
 // This is equivalent to `oc get secrets -n openshift-monitoring alertmanager-main`.
 // It specifically extracts the .data "alertmanager.yaml" field, and loads it into a resource
 // of type Config, enabling it to be marshalled and unmarshalled as needed.
-func getAlertManagerConfig(r *ReconcileSecret, request *reconcile.Request) alertmanager.Config {
```

Comment:
After this renaming this function the comment above is still referring to the previous name.